AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Core/SQLStatementTests.swift

```diff
@@ -198,6 +259,34 @@ class SQLStatementTests: XCTestCase {
         XCTAssertEqual(variables[6] as? String, "title")
     }
 
+    /// - Given: a `Model` instance
+    /// - When:
+    ///   - the model has a composite pk
+    /// - Then:
+    ///   - check if the generated SQL statement is valid
+    ///   - check if the variables match the expected values
+    func testInsertStatementFromModelWithCompositePK() {
+
+        let modelId = "the-id"
+        let dob = Temporal.DateTime.now()
+        let model = ModelCompositePk(id: modelId,
+                                     dob: dob,
+                                     name: "the-name")
+        let statement = InsertStatement(model: model, modelSchema: model.schema)
+
+        let expectedStatement = """
+        insert into "ModelCompositePk" ("@@primaryKey", "id", "dob", "createdAt", "name", "updatedAt")
+        values (?, ?, ?, ?, ?, ?)
+        """
+        XCTAssertEqual(statement.stringValue, expectedStatement)
+
+        let variables = statement.variables
+        XCTAssertEqual(variables[0] as? String, "\(modelId)#\(dob.iso8601String)")
```

Comment:
shouldn't the `modelId` be inside quotes like " \ " \(modelId) \ " " ?

Reply:
yes, updated the logic since we've reached consensus across platform.